src/Components/Web.JS/src/Rendering/BrowserRenderer.ts

```diff
@@ -58,7 +58,7 @@ export class BrowserRenderer {
       }
     }
 
-    const ownerDocument = getClosestDomElement(element).ownerDocument;
+    const ownerDocument = getClosestDomElement(element) ? getClosestDomElement(element).ownerDocument : null;
```

Comment:
Just to avoid evaluating `getClosestDomElement(element)` twice, could we do:

```suggestion
    const ownerDocument = getClosestDomElement(element)?.ownerDocument;
```